### ISSUE TITLE
potential/growth.sh fails in 6.1 branch but pass in master branch

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#3737]
Perhaps a commit to master should be backported to 6.1?

**Comment by PaulWessel** [comment]
Hm, maybe.  I worry that I added many new things so not sure if this needs to be 6.1.

**Comment by PaulWessel** [comment]
Maybe if you could try it and if it causes trouble we can undo?

**Comment by seisman** [comment]
growth.sh was recently updated in PR #3733, but I can't backport it 6.1 branch, due to some conflicts.

**Comment by PaulWessel** [comment]
OK, perhaps I fix this manually in 6.1

**Comment by seisman** [comment]
Here is the error message:
```
1/2 Test #547: potential/growth.sh ..............***Failed    4.02 sec
Set GMT_SESSION_NAME = 55058
grdseamount [WARNING]: No contribution made for time 10 yr
grdseamount [WARNING]: No contribution made for time 0.75 yr
grdseamount [WARNING]: No contribution made for time 0.5 yr
grdseamount [WARNING]: No contribution made for time 0.25 yr
grdseamount [WARNING]: No contribution made for time 0 yr
psxy [ERROR]: Option -G parsing failure.  Correct syntax:
	-G<fill>   Specify <fill> as one of:
	   1) <gray> or <red>/<green>/<blue>, all in the range 0-255;
	   2) #rrggbb, all in the range 0-255 using hexadecimal numbers;
	   3) <c>/<m>/<y>/<k> in range 0-100%;
	   4) <hue>-<sat>-<val> in ranges 0-360, 0-1, 0-1;
	   5) any valid color name;
	   6) P|p<pattern>[+b<color>][+f<color>][+r<dpi>];
	      Give <pattern> number from 1-90 or a filename, optionally add +r<dpi> [0].
	      Optionally, use +f<color> or +b<color> to change fore- or background colors (no <color> sets transparency).
	   For PDF fill transparency, append @<transparency> in the range 0-100 [0 = opaque].
ERROR: gmtest:58
memtrack errors: 0
exit status: 1
ERROR: gmtest:58
memtrack errors: 0
exit status: 1
ERROR: growth.sh:28
memtrack errors: 0
exit status: 1
```

**Comment by seisman** [comment]
@PaulWessel This issue has to be fixed before releasing 6.1.1.
